Challenge-4/Bot_1/src/constants.ts

```diff
@@ -37,3 +36,14 @@ export const ASSET_INFO = [
   `function getAssetInfo(uint8 i) public view returns (uint8 offset, address asset, address priceFeed, uint64 scale, uint64 borrowCollateralFactor, uint64 liquidateCollateralFactor, uint64 liquidationFactor, uint128 supplyCap)`,
 ];
 
+export const RESULTS_ARRAY = [
```

Comment:
```suggestion
export const EVENTS_ARRAY = [
```
Small suggestion to rename it so it is more clear. 🙂